ci/dev/watch.ts

```diff
@@ -150,7 +142,7 @@ class Watcher {
         startingVscode = true
       } else if (startingVscode && line.includes("Finished compilation")) {
         if (startedVscode) {
-          bundle.then(restartServer)
+          restartServer()
```

Comment:
Do we need to call `bundleBrowserCode(browserFiles)` here or no?

Reply:
Nope, we only need the one on line 157 since that's for code-server's code.